### DISCUSSION TITLE
Kanban views

### DISCUSSION THREAD

**Discussion by kgodey** [mathesar-foundation/mathesar#2283] [Feature Ideas]
We'd like to support the ability to display a table or exploration as a Kanban board.

---
*If you'd like us to build this feature, please comment below with more details about how you plan to use this functionality.*

**Comment by steve-taylor** [comment]
The most obvious use case is to use Mathesar as an alternative to Jira, grouping tables by ticket status.

Jira is OK, but teams often have complex workflows and would like something that can fit in with their process, not the other way around. Airtable would fill this niche perfectly if it didn't have the world's most expensive database rows.

**Reply by kgodey** [reply]
Thanks for the comment, @steve-taylor! Are there any automations or integrations with other tools to make Mathesar useful as an alternative to Jira for your team, or would just a kanban view suffice?
